src/figuresClasses.ts

```diff
@@ -1,19 +1,76 @@
 export interface Figure {
-
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'blue' | 'green';
```

Comment:
you can create type for `shape` and `color`

`type Shape = 'triangle' | 'circle' | 'rectangle'` and just use everywhere `Shape` instead of `Figure['shape']`